packages/sources/ion.au/src/transport/total_reserve.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/total_reserve'
+
+export interface ResponseSchema {
+  total_reserve: number
+  errorMessage?: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: '/',
+          headers: {},
+          params: {},
+        },
+      }
+    })
```

Comment:
I think Alejo's suggestions apply to all of these returns that use `params.map`. We could simplify this too like  
```suggestion
    return {
      params,
      request: {
        baseURL: config.API_ENDPOINT,
        url: '/',
        headers: {},
        params: {},
      },
    }
```